config/global.ini.php

```diff
@@ -45,6 +45,9 @@
 ; Matomo should work correctly without this setting but we recommend to have a charset set.
 charset = utf8
 
+; disable dynamic row by default
+disable_dynamic_row_format = true
```

Comment:
fyi here we usually use `1` for enabled and `0` for disabled.

Also might be more readable by naming it `enable_dynamic_row_format = 0` but not too important